streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -782,10 +836,35 @@ private void checkIfUnexpectedUserSpecifiedConsumerConfig(final Map<String, Obje
      * @param groupId      consumer groupId
      * @param clientId     clientId
      * @return Map of the consumer configuration.
+     * @Deprecated use {@link StreamsConfig#getMainConsumerConfigs(String, String)}
      */
+    @Deprecated
     public Map<String, Object> getConsumerConfigs(final String groupId,
                                                   final String clientId) {
-        final Map<String, Object> consumerProps = getCommonConsumerConfigs();
+        return getMainConsumerConfigs(groupId, clientId);
+    }
+
+    /**
+     * Get the configs to the {@link KafkaConsumer main consumer}.
+     * Properties using the prefix {@link #MAIN_CONSUMER_PREFIX} will be used in favor over their non-prefixed versions
```

Comment:
nit: ` .. over the properties prefixed with {@link #CONSUMER_PREFIX} and the non-prefixed versions (read the override precedence ordering in {@link #MAIN_CONSUMER_PREFIX)..` ditto below.